chapters/revisions.tex

```diff
@@ -2256,7 +2256,11 @@ \subsection{Main Changes in Modelica 1.4}\label{main-changes-in-modelica-1-4}
   When copying or moving an encapsulated class, at most the import
   statements in this class have to be changed.
 \item
-  Refined when-clause: The \lstinline!nondiscrete! keyword is removed, equations in when-clauses must have a unique variable name on left hand side variable and the exact mapping of when-clauses to equations is defined.  As a result, when-clauses are now precisely defined without referring to a sorting algorithm and it is possible to handle algebraic loops between when-clauses with different conditions and between when-clauses and the continuous-time part of a model.  The \lstinline!discrete! keyword is now optional, simplifying the library development because only one type of connector is needed and not several types which do contain or do not contain the discrete prefix on variables.  Additionally, when-clauses in algorithm sections may have elsewhen-clauses which simplifies the definition of priorities between when-clauses.
+  Refined when-clause:
+  The \lstinline!nondiscrete!\index{nondiscrete@\robustinline{nondiscrete} (removed keyword)} keyword is removed, equations in when-clauses must have a unique variable name on left hand side variable and the exact mapping of when-clauses to equations is defined.
```

Comment:
In reply to @beutlich:

> Not sure, if you really want to have removed content like **nondiscrete** in the index.

It can easily be removed.  It ended up in the index because it is still highlighted as a keyword, but to me a more important reason to have it in the index is that a user might encounter old code with `nondiscrete`, and placing it in the index is the best way I can think of for helping the user understand its current (non-existent) role.

Let's see if we can get more input to this discussion…